Added org-tree-slide recipe

takaxp opened the pull request:
This is a presentation tool for org-mode.
Please integrate into MELPA.
Thanks.

purcell commented:
Thanks.

Please remove the `:files` spec from the recipe. The default includes `*.el` and is fine here. Ideally, amend the commit and force-push it back here to update the pull request cleanly.

I strongly recommend avoiding the `ots-` prefix for identifiers, because it may clash with other future packages. It would be [more idiomatic](https://www.gnu.org/s/emacs/manual/html_node/elisp/Coding-Conventions.html) to use `org-tree-slide-`, or `org-tree-slide--` for "private" symbols.

-Steve

takaxp commented:
Hi Steve,
Thank you for your kind instruction. I removed the files property and also updated my code as you suggested.
Thanks in advance.

Best,
Takaaki

purcell commented:
Thanks. Applied in 6160c25.
